src/apps/pillarx-app/components/EditorialTile/EditorialTile.tsx

```diff
@@ -0,0 +1,115 @@
+            const mimeType = mime.getType(editorialDisplay.media);
+
+            if (mimeType?.includes('image')) {
+                return (
+                    <img src={editorialDisplay.media} className='rounded-lg object-cover object-center desktop:h-full tablet:h-full mobile:w-full' data-testid='editorial-tile-image' />
+                );
+            }
+
+            if (mimeType?.includes('video')) {
+                return (
+                    <video className='rounded-lg w-full' preload='none' autoPlay loop muted controls playsInline data-testid='editorial-tile-video'>
+                        <source src={editorialDisplay.media} type={mimeType} />
+                    </video>
+                );
+            }
+
+            if (mimeType?.includes('audio')) {
+                return (
+                    <audio className='w-full' preload='none' controls data-testid='editorial-tile-audio'>
+                        <source src={editorialDisplay.media} type={mimeType} />
+                    </audio>
+                );
+            }
+        }
+
+        return null;
+    };
+
+    const editorialDate = editorialDisplay?.timestamp && new Date(editorialDisplay.timestamp * 1000);
+
+    if (!data || isDataLoading) {
+        return (
+            <TileContainer className='flex-col desktop:p-10 desktop:pt-[30px] tablet:p-5 mobile:p-0 mobile:bg-[#1F1D23]'>
+                <SkeletonLoader $height='24px' $width='150px' $marginBottom='10px' $radius='6px' />
+                <div data-testid="editorial-tile-loading" className='flex mobile:flex-col bg-medium_grey rounded-2xl p-2 gap-4'>
+                    <div className='flex desktop:basis-2/5 tablet:basis-2/5 mobile:basis-full mobile:w-full'>
+                        <SkeletonLoader $height='250px' $width='100%' $radius='6px' />
+                    </div>
+                    <div className='flex flex-col justify-between desktop:basis-3/5 tablet:basis-3/5 mobile:basis-0 mobile:w-full desktop:px-4 tablet:px-4 mobile:px-0'>
+                        <div className='flex flex-col gap-2 desktop:mt-14 tablet:mt-8 mobile:mt-0'>
+                            <SkeletonLoader $height='26px' $width='50px' $radius='6px' />
+                            <SkeletonLoader $height='30px' $width='200px' $radius='6px' />
+                        </div>
+                        <div className='mb-[18px] mt-4 flex justify-between'>
+                            <SkeletonLoader $height='24px' $width='80px' $radius='6px' />
+                            <SkeletonLoader $height='24px' $width='60px' $radius='6px' />
+                        </div>
+                    </div>
+                </div>
+            </TileContainer>
+        );
+    }
+
+    return (
+        <TileContainer id='editorial-tile' className='flex-col desktop:p-10 desktop:pt-[30px] tablet:p-5 mobile:p-0 mobile:bg-[#1F1D23]'>
+            <a href={editorialDisplay?.href} target="_blank" rel="noreferrer">
```

Comment:
If there is no `href` this should not be clickable or render the `a` tag. You could fall back to a `onClick` + `cursor: pointer` which would allow you to check the variable or conditionally render the tag